### PR TITLE
Proposal to add BioCreative VII Track 5 LitCovid dataset (#400)

### PR DESCRIPTION
**Pull request by qanastek**
### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Comment by qanastek** [comment]
Commit https://github.com/bigscience-workshop/biomedical/issues/400

**Comment by qanastek** [comment]
Just have an unknown license.

File: biodatasets/litcovid/litcovid.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
```
Comment (leonweber) [inline biodatasets/litcovid/litcovid.py:25]: The correct reference should be https://biocreative.bioinformatics.udel.edu/media/store/files/2021/TRACK5_pos_1_BC7_submission_221.pdf

**Review by leonweber — CHANGES_REQUESTED** [review]

**Comment by leonweber** [comment]
I think this is ready to be merged after the reference has been changed

**Comment by leonweber** [comment]
We should rename the dataset to `bc7_litcovid` to make it clear that this is the BC7 shared task dataset and not the continually updated litcovid.

**Review by leonweber — CHANGES_REQUESTED** [review]
We should rename the dataset to `bc7_litcovid` to make it clear that this is the BC7 shared task dataset and not the continually updated litcovid.

File: biodatasets/litcovid/litcovid.py (hunk 1 above)
Comment (qanastek) [reply biodatasets/litcovid/litcovid.py:25]: Ok! changed.

**Comment by qanastek** [comment]
Commit 5466de407db6211024ad656eed7508364d81be2b

**Review by leonweber — APPROVED** [review]
LGTM, thanks for your contribution! 🎉

**Pull request merged**